docs/DeveloperGuide.md

```diff
@@ -303,25 +363,57 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
   Use case ends.
 
 * 3a. The given index is invalid.
+    * 3a1. Duke The Market shows an error message.
+
+      Use case resumes at step 3.
+
+
+**UC04: Export email addresses of customers to a file**
+
+**MSS**
+1.  User requests to export to a specified file the email address of customers who attended a specific event.
+2.  Duke The Market exports the customers’ email addresses to the specified file
+
+    Use case ends.
+
+
+**UC05: Retrieve customers who may be interested in an upcoming event**
+
+**MSS**
+1. User requests to list events
+2. Duke The Market shows a list of events
+3. User requests to show customers who may be interested in an upcoming event in the list
+4. Duke The Market shows the customers who may be interested in that upcoming event
+5. User requests to export the customers' email addresses to a specified file (UC04)
```

Comment:
I think step 5 should be an extension as written in the google docs? Or do you want it to be part of the MSS?

Also I believe the notation to show inclusion is to underline the included use case. (use `<ins>`text`</ins>` to underline)

```suggestion
5. User requests to <ins>export the customers' email addresses to a specified file (UC04)</ins>
```

Reply:
Ah yes, my bad!